packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -1,36 +1,32 @@
 <template>
   <picture ref="image" class="x-picture x-result-picture" data-test="result-picture">
-    <component
-      :is="animation"
+    <img
       v-if="!hasImageLoaded && !hasAllImagesFailed"
```

Comment:
Because of the bug in Safari about empty SRCs reported by Wences and Ivan,
Could we remove the `hasAllImagesFailed` computed property and just use `!imageSrc` instead?. Similar approach for below changes. 
```suggestion
      v-if="!hasImageLoaded && imageSrc"
```